src/momento-redis-adapter.ts

```diff
@@ -824,6 +830,14 @@ export class MomentoRedisAdapter
       return 'OK';
     }
   }
+
+  pipeline(commands?: Command[][]): ChainableCommander {
+    const pipeline = new Pipeline(this);
+    if (Array.isArray(commands)) {
```

Comment:
Let's also document this line here is for type safety and mirrors the ioredis implementation.